src/main/java/net/fabricmc/meta/utils/MinecraftLauncherMeta.java

```
@@ -80,6 +90,25 @@ public String getTime() {
 		public String getReleaseTime() {
 			return releaseTime;
 		}
+
+		public String getMajorVersion() {
+			String json = null;
+			try {
+				json = IOUtils.toString(new URL(url), StandardCharsets.UTF_8);
+			} catch (IOException e) {
+				e.printStackTrace();
```

This is just going to fail harder on the next line.